### DISCUSSION TITLE
Wrong display type of BT-106-Procedure field in notice type 18

### DISCUSSION THREAD

**Discussion by tender-systems** [OP-TED/eForms-SDK#1078] [Q&A] [answered]
The field BT-106-Procedure has a COMBOBOX display type for notice type 18 in 1.8.2 SDK, but in 1.13 it has a TEXTBOX display type. The BT-106-Procedure is still indicator in the fields.json. Please could you check it? Thanks in advance. Jiri

**Comment by encampaen** [comment, marked as answer]
Dear Jiri,
We wil change it back to COMBOBOX in the next version of the SDK
Best regards
Enrico
